packages/react-core/package.json

```diff
@@ -40,6 +40,7 @@
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "scripts": {
     "build:umd": "rollup -c --environment IS_PRODUCTION",
+    "build:fed:packages" : "node ../../scripts/build-single-packages.js --config build-single-packages.json",
```

Comment:
I'm curious about the name of the script here. Is it called "build:fed" only because federated plugins want the feature? Is it possible non-federated plugins might want the tree-shaking abilities that comes with this build as well?

Reply:
updated it to build:single:packages